Library/ViewModels/PaymentMethodsViewModel.swift

```diff
@@ -195,6 +207,9 @@ public final class PaymentMethodsViewModel: PaymentMethodsViewModelType,
     )
     .map { $0.localizedDescription }
 
+    self.shouldCancelPaymentSheetAppearance <~ self.errorLoadingPaymentMethodsOrSetupIntent.signal
```

Comment:
MI wonder if there is a way to assign all of these signals to `shouldCancelPaymentSheetAppearance` in one line 🤔 

Or maybe we can group these lines together so that it's a little easier to see all of the states that `shouldCancelPaymentSheetAppearance` reacts to. Might make it easier to debug in the future. Not a big deal